packages/core/src/modules/proofs/protocol/v1/messages/V1PresentationAckMessageV1.ts

```diff
@@ -0,0 +1,20 @@
+import type { AckMessageOptions } from '../../../../common'
+
+import { Equals } from 'class-validator'
+
+import { AckMessage } from '../../../../common'
+
+export type PresentationAckMessageOptions = AckMessageOptions
```

Comment:
Not sure if this adds anything, AckMessageOptions is fine as type in the constructor.